.travis.yml

```diff
@@ -1,18 +0,0 @@
-sudo: false
-language: java
-jdk:
-- oraclejdk7
-env:
-  global:
-  - secure: IE/mfL1X/GtHnU5HtxL24u/SSkoTXnBBHbtfIpyjYwgbzGhPuq9qn9nEOBNb43ePaPGO+PDMz9sxDxHSIwBgbomEmq3dvQ9j+2jOx7MhmtCVr8Jz9x0xhug4cVPhaMVUZDa+z3qfSzbDbdtRKvuAzWHhwnbS9gQaGn1oIR2++5tXGZk+LVyvDO5/wbR6EcMxDQf0ZmUde/q0QtDmkIinS5r09ydUhsJE1wCUzyiOhx/kErnGmGD6buI/AgiZeGsGouonCykjrlNkoZkgmHobu/eCfbZPxUY9AkoJxORiLXZQ4GYH3KAvJlo5OZo99W8nw7wgnKZLJVkQD/sZzz3BCNlFQj41MroGUECm+fAq1hCDkXM1AmyFj9ZDQGyeb5+uEhf37mCN0TnUP6PNTTr+g/CqhQK67GDXM0i8LLKpEa/KtRuTtYm+gQ1VWJFYnxTx121XpEW8RKooqMbnBtOGngZqDqYb/8UjXg97logHThPMzpx3pkmDKlI+eB6wKi8VuBcMMIMU7K/pqUyWMqjK+JTAorPHJH17pCywsqsw0XghrIxjBjfTi7v8i9JxmV+9IadiaHhBj1WxJT3WnNEt7WEp4Q0ZxAL5ToiHT25ga55+9MHP9rJnqyunXon0q6MUoO3ubUmHU3wukCbakvZb7h8fEuCtMdzfA0TIZ7EsDIE=
```

Comment:
I imagine this needs to transfer over to Circle config, but not sure how to do so